passkey-api/next-auth.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Next.js with Auth.js
+---
+
+If you already have a Next.js set up with Auth.js, you can easily passkeys to your app using `@teamhanko/passkeys-next-auth-provider`.
```

Comment:
```suggestion
If you already have a Next.js app set up with Auth.js, you can easily add passkeys to your app using `@teamhanko/passkeys-next-auth-provider`.
```

Reply:
Wow, no idea what happened here. Thank you